package.json

```
@@ -12,7 +14,8 @@
     "test:update": "ava --update-snapshots",
     "test": "ava",
     "lint": "eslint .",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release --dry-run",
```

Can we revert that change please?

```suggestion
    "semantic-release": "semantic-release",
```

---

My mistake, fixed it :)